cassdegrees/static/js/rules.js

```diff
@@ -3,6 +3,7 @@
 // Translation table between internal names for components and human readable ones.
 const COMPONENT_NAMES = {
     'subplan': "Subplan",
+    'subject_area_specific': "Subject-Area Specific Units",
```

Comment:
```suggestion
    'subject_area': "Subject-Area Units",
```

Replace use of 'specific' here, as I think it unnecessarily complicates the terminology (NOTE: must accept all changes or this won't work)